src/openloco/interop/interop.hpp

```diff
@@ -26,7 +26,7 @@ namespace openloco::interop
     {
         union
         {
-            int32_t eax;
```

Comment:
Does this fill `eax` with `0xCCCCCCCC`, or `0x000000CC`?

Reply:
derp, of course I did it wrong.

Reply:
in that case, perhaps I should use something like `CAFEBABE`, `DEADBEEF`, etc?